pkg/job/associator/associator_test.go

```diff
@@ -283,6 +289,25 @@ func TestAssociator(t *testing.T) {
 			expectedSkip:     false,
 			expectedResource: ecsService2,
 		},
+		{
```

Comment:
Can you please add a similar test case for the "max dimensions associator"? It's a new algorithm currently behind a feature flag, more info [here](https://github.com/nerdswords/yet-another-cloudwatch-exporter/blob/master/docs/feature_flags.md#new-associator-algorithm).

You could create a file named `pkg/job/maxdimassociator/associator_memorydb_test.go` like [this one](https://github.com/nerdswords/yet-another-cloudwatch-exporter/blob/master/pkg/job/maxdimassociator/associator_ec2_test.go).

Reply:
@cristiangreco added, please take a look